src/main/java/com/cegeka/academy/service/dto/UserChallengeDTO.java

```diff
@@ -0,0 +1,100 @@
+package com.cegeka.academy.service.dto;
+
+import com.cegeka.academy.domain.Challenge;
+import com.cegeka.academy.domain.Invitation;
+import com.cegeka.academy.domain.User;
+
+import java.util.Date;
+
+public class UserChallengeDTO {
+
+    private Long id;
+    private User user;
+    private Invitation invitation;
+    private Challenge challenge;
```

Comment:
Un DTO nu poate contine atribute de tipul entităților. Dam in aceeasi problema. User Invitation si Challenge nu au ce cauta aici. Ar trebui să definești dto uri si pentru acestea. De asemenea intr un dto nu trebuie sa pui toate câmpurile din entitate daca nu sunt necesare pentru cliebt